recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.12" %}
+
+package:
+  name: uwsgi
+  version: '{{ version }}'
+
+source:
+  fn: "uwsgi-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/u/uWSGI/uwsgi-{{ version }}.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+  patches:
+    - darwin.patch   # [osx]
+    - darwin2.patch  # [osx]
+    # ^^ patch darwin2 should be able to be deleted on the next version pump
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+    skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - gcc      # [unix]
+    # - jansson
+    - libxml2  # [unix]
+    - openssl 1.0.2*
+    - pcre
+    - yaml
+    - zlib 1.2*
+  run:
+    - python
+    # - jansson
+    - libxml2  # [unix]
+    - openssl 1.0.2*
```

Comment:
It's not clear to me that we need to pin this that tightly. According the [OpenSSL FAQ]( https://www.openssl.org/docs/faq.html#MISC8 ) they claim that ABI compatibility is preserved over what they call the minor version or third number here. So, I think we only need to pin this to `1.0.*` instead to preserve flexibility in the future.